Kickstarter-iOS/Views/Cells/CommentRemovedCell.swift

```diff
@@ -0,0 +1,130 @@
+  private lazy var rowStackView: UIStackView = {
+    UIStackView(frame: .zero)
+  }()
+
+  private lazy var commentCellHeaderStackView: CommentCellHeaderStackView = {
+    CommentCellHeaderStackView(frame: .zero)
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.setupConstraints()
+    self.bindStyles()
+    self.configureViews()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.rootStackView
+      |> commentCellRootStackViewStyle
+
+    _ = self.rowStackView
+      |> rowStackViewStyle
+
+    _ = self.commentLabel
+      |> \.attributedText .~ attributedTextCommentRemoved()
+      |> \.lineBreakMode .~ .byWordWrapping
+      |> \.numberOfLines .~ 0
+      |> \.adjustsFontForContentSizeCategory .~ true
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(value: DemoComment) {
+    self.commentCellHeaderStackView.configureWith(comment: value)
+  }
+
+  private func configureViews() {
+    self.rowStackView.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+    _ = ([self.commentCellHeaderStackView, self.rowStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.infoImageView, self.commentLabel], self.rowStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+
+  private func setupConstraints() {
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    NSLayoutConstraint.activate([
+      self.infoImageView.widthAnchor.constraint(equalToConstant: Styles.grid(4)),
+      self.infoImageView.heightAnchor.constraint(equalToConstant: Styles.grid(4))
+    ])
+  }
+}
+
+// MARK: Styles
+
+private let rowStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.alignment .~ .top
+    |> \.spacing .~ Styles.grid(1)
+}
+
+// MARK: - Functions
+
+// TODO: Internationalized in the near future.
+
+private func attributedTextCommentRemoved() -> NSAttributedString {
+  let regularFontAttribute = [
+    NSAttributedString.Key.font: UIFont.ksr_callout(),
+    NSAttributedString.Key.foregroundColor: UIColor.ksr_support_400
+  ]
+  let coloredFontAttribute = [
+    NSAttributedString.Key.font: UIFont.ksr_callout(),
+    NSAttributedString.Key.foregroundColor: UIColor.ksr_create_700
+  ]
+
+  let attributedString = NSMutableAttributedString(
+    string: localizedString(
+      key: "This_comment_has_been_removed_by_Kickstarter",
+      defaultValue: "This comment has been removed by Kickstarter. "
+    ),
+    attributes: regularFontAttribute
+  )
+  let learnMorAttributedString = NSMutableAttributedString(
```

Comment:
Will this be made into a link in a different PR?

Reply:
Typo in the name of the variable here

Reply:
I was thinking it should go in a different PR, but seeing that it won't take much time to fix, I'll be adding it to this PR

Reply:
So I think this would need to be in a different ticket, there's currently a ticket raised for localization which I believe @singhhari is assigned to.
The project has a template to work with links from text and it seems to be tied with localized string.

Reply:
I'll update the localization ticket to include this here but leave a TODO/FIXME so I can search for it